Sources/Core/API/TextureManager.swift

```diff
@@ -13,6 +13,8 @@ public final class TextureManager {
     public var imageLoader: TextureImageLoader
     /// The default scale when loading textures (default = the main screen's scale)
     public var defaultScale: Int = Int(Screen.mainScreenScale)
+    /// The default format when loading textures (default = PNG)
+    public var defaultFormat: TextureFormat = .png
```

Comment:
Good call 👍 I didn't think about it when reviewing earlier, but this fits very nicely into the same pattern as used for scale 👌